Source/Core/Core/CMakeLists.txt

```diff
@@ -766,4 +768,4 @@ endif()
 if(USE_RETRO_ACHIEVEMENTS)
   target_link_libraries(core PRIVATE rcheevos)
   target_compile_definitions(core PRIVATE -DUSE_RETRO_ACHIEVEMENTS)
-endif()
\ No newline at end of file
+endif()
```

Comment:
Interesting that our linter didn't complain about the lack of a newline at the end of the file before. (Adding one is correct, so you don't need to do anything about this.)